### PR TITLE
These new html files have the logo updated

### PR DESCRIPTION
**Pull request by TanishaAgrawal**

### CODE REVIEW

**Comment by TanishaAgrawal** [comment]
Invoice

File: src/com/hsbc/scheduler/Schedular.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+			
+			//call the function to be executed here
+			System.out.println("CONSTANT---" + midnight + "----" + initialDelay + "-----" + now);
+			System.out.println("Repeat task: " + LocalDateTime.now());
+		},0,10000,TimeUnit.MILLISECONDS );
+		
+		//initial period, delay between every execution
+		
+	}
+	public void expire() throws OrderNotFoundForEmployee, ProductNotFoundException
```
Comment (ujjwalmishra21) [inline src/com/hsbc/scheduler/Schedular.java:49]: These functions aren't getting called anyhere.

File: src/com/hsbc/controllers/GetInvoice.java [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+		PrintWriter out = response.getWriter();
+		ArrayList<Product>productList=new ArrayList<>();
+		/*productList.add(new Product(2,"Top", 300,1, null, null));
+		productList.add(new Product(3,"shirt", 300,2, null, null));
+		productList.add(new Product(4,"e", 300,3, null, null));
+		productList.add(new Product(5,"op", 300,1, null, null));
+		productList.add(new Product(1,"eop", 300,2, null, null));
+		*/
+			
+			 try {
```
Comment (ujjwalmishra21) [inline src/com/hsbc/controllers/GetInvoice.java:42]: No need of creating db connection here

File: src/com/hsbc/service/InvoiceService.java [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+		obj1.setGstTypeId(0);
+		obj1.setGstAmount(generateGST(obj3.getTotalOrderValue()));
+		double totalInvoiceValue=generateGST(obj3.getTotalOrderValue())+getShippingCost(obj3.getProducts(),obj3.getTotalOrderValue());
+		obj1.setTotalInvoiceAmount(totalInvoiceValue);
+		obj1.setStatus(true);
+		//Invoice obj=empDAO.getInvoiceByOrderId(orderId);
+		return obj1;
+	}
+	public int generateInvoiceId()
+	{
```
Comment (ujjwalmishra21) [inline src/com/hsbc/service/InvoiceService.java:33]: Delete this method

File: src/com/hsbc/service/InvoiceService.java [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+	}
+	public double getShippingCost(ArrayList<Product> products, double totalOrderValue )
+	{
+		double shippingCharges=0;
+		int i=0;
+		
+		if(totalOrderValue>1000000) {
+			shippingCharges=0;
+		}
+		else
```
Comment (ujjwalmishra21) [inline src/com/hsbc/service/InvoiceService.java:50]: Comment

File: src/com/hsbc/controllers/GetInvoice.java [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+import javax.servlet.http.HttpServletResponse;
+import com.hsbc.dao.EmployeeDAOImpl;
+import com.hsbc.models.Invoice;
+import com.hsbc.models.Product;
+
+
+@WebServlet("/getInvoice")
+public class GetInvoice extends HttpServlet {
+	private static final long serialVersionUID = 1L;
+
```
Comment (ujjwalmishra21) [inline src/com/hsbc/controllers/GetInvoice.java:25]: private InvoiceService invoiceService;

File: src/com/hsbc/controllers/GetInvoice.java [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+import com.hsbc.models.Product;
+
+
+@WebServlet("/getInvoice")
+public class GetInvoice extends HttpServlet {
+	private static final long serialVersionUID = 1L;
+   
+    public GetInvoice() {
+        super();
+        // TODO Auto-generated constructor stub
```
Comment (ujjwalmishra21) [inline src/com/hsbc/controllers/GetInvoice.java:28]: invoiceService = new InvoiceService();

File: src/com/hsbc/controllers/GetInvoice.java [hunk 7]
```diff
@@ -0,0 +1,99 @@
...
+				e.printStackTrace();
+			} catch (SQLException e) {
+				// TODO Auto-generated catch block
+				e.printStackTrace();
+			} 
+			 
+			//int orderId=request.getParameter("name of attribute");
+			//ArrayList<Product>productList=new ArrayList<>();
+			//productList=new OrderDetailService().getProductList();
+			int orderID=Integer.parseInt(request.getParameter("orderId"));
```
Comment (ujjwalmishra21) [inline src/com/hsbc/controllers/GetInvoice.java:56]: invoiceService.getInvoiceByOrderId(orderId);

**Review by ujjwalmishra21 — APPROVED** [review]

**Pull request merged**